lib/ex_statsd.ex

```diff
@@ -271,6 +289,20 @@ defmodule ExStatsD do
     ] |> IO.iodata_to_binary
   end
 
+  defp packet({:event, title, text, opts}, _namespace, tags, _sample_rate) do
+    [
+      "_e",
+      "{#{title |> String.length},#{text |> String.length}}",
```

Comment:
text needs to be escaped from new lines like `String.replace(text, "\n", "\\n")`